<launch>

  <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find gazebo_sfm_plugin)/media/models"/>
  <env name="GAZEBO_RESOURCE_PATH" value="${GAZEBO_RESOURCE_PATH}:$(find gazebo_sfm_plugin)/media/models"/>
  
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find gazebo_sfm_plugin)/worlds/cafe_tfm.world"/>
  </include>

    <node name="task_coordinator" 
        pkg="gazebo_sfm_plugin" 
        type="task_coordinator_node"
        output="screen"/>

</launch>
